platform/android/scripts/generate-style-code.list

```diff
@@ -0,0 +1,36 @@
+platform/android/src/style/light.hpp
```

Comment:
Could we move all generated files to `/generated` folders or rename them `<filename>.gen.<ext>`. 

It would remove the need for the `generate-style-code-list` files, and (in addition to the comments at the top of the file) make it obvious when generated files are being viewed/modified.

Reply:
For iOS and macOS, it wouldn’t be practical to rename the files, since the header file names are exposed as-is to developers and the Markdown file names show up in URLs in the API reference. However, moving generated files to several /generated folders would be straightforward, since Xcode’s project structure is an abstraction of the file system anyways. Note that the generated guides to have to live more or less where they do now.